src/controllers/RequestController.js

```diff
@@ -66,6 +66,43 @@ class RequestController {
       return HelperMethods.serverError(res);
     }
   }
+
+  /**
+  * Reject a Request
+  * Route: PATCH: /request
+  * @param {object} req - HTTP Request object
+  * @param {object} res - HTTP Response object
+  * @return {res} res - HTTP Response object
+  * @memberof RequestController
+  */
+  static async rejectRequest(req, res) {
+    try {
+      const { role } = req.decoded;
+      const { id } = req.params;
+
+      if (role === 'Manager') {
```

Comment:
Your logic doesn't check if request is from Manager's direct report.

Reply:
Hello Chris, the logic is a condition that checks if the user is a manager, he is able to reject a request. Outside the if statement on line 97 returns a statement if the user is not a manager.

Reply:
Yes.I agree with the logic. But an extra logic has to be there to ensure that not every manager can reject request made by another manager's direct request. That's why there is a column called lineManager

Reply:
I am working on a similar feature. You can check => https://www.pivotaltracker.com/story/show/167727641

Reply:
Yeah, but in this case, there is no defined organization structure that we can use here. How do I check if a particular user is another's user's line manager. From my understanding of the user roles, there are 5 user roles as provided in the PT board.

Reply:
> How do I check if a particular user is another's user's line manager

Get all requests include User=> where user.line manager === managerId